### PR TITLE
Run list_unsupported.sh in CI

### PR DESCRIPTION
**Pull request by chrisr-diffblue**

### CODE REVIEW

**Comment by chrisr-diffblue** [comment]
I've got some ideas around that - but yeah, nothing will be perfect or zero maintenance here. This PR is mostly just a playground for me to try out some ideas right now :-) hence the "do not merge" "do not review" labels :-)

**Comment by martin-cs** [comment]
Sounds both good and fair.

File: experiments/list_unsupported.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ include_path=""
 DIR=`dirname "$0"`
 file_name=$(basename "$1")
 
-for foldername in $(find ${path} -type d -name "*"); do
+for foldername in $(find ${path} -type d -name "*" | LC_ALL=posix sort ); do
```
Comment (hannes-steffenhagen-diffblue) [inline experiments/list_unsupported.sh]: Can you add an explanation for what the `LC_ALL=posix` is for to the commit message?

File: gnat2goto/COMPILING.md [hunk 2]
```diff
@@ -68,6 +68,29 @@ through to final analysis by CBMC.
...
+following commands:
+
+- `$ export PATH=/opt/gnat/bin:/path/to/gnat2goto/bin:${PATH}`
+- `$ export GPR_PROJECT_PATH=/opt/gnat/lib/gnat`
+- `$ cd gnat2goto/experiments`
+- `$ ./CI-list_unsupported.sh`
+
+This script will checkout a number of Ada projects, then for each project it will
+run gnat2goto and summarize the unsupported features used by the project. The
+summary is then compared against a 'golden' reference summary.
```
Comment (hannes-steffenhagen-diffblue) [inline gnat2goto/COMPILING.md]: Might make the 'golden' here a link to an explanation, like [this](https://ro-che.info/articles/2017-12-04-golden-tests)?

Comment (chrisr-diffblue) [reply gnat2goto/COMPILING.md]: I thought this was a relatively well known term (it is, afterall, the same type of testing we use for things like CBMC regression tests, gnat2goto regression tests, etc), but I can dig out some sort of reference if you want for a followup PR.

Comment (NlightNFotis) [reply gnat2goto/COMPILING.md]: To be honest, I didn't know it was an established term. I mean, I could more or less imagine what it meant, but if it wasn't for Hannes' link I wouldn't know.

Comment (chrisr-diffblue) [reply gnat2goto/COMPILING.md]: OK, I'll add a reference then :-)

**Pull request merged**